### PR TITLE
#1202 

### PR DESCRIPTION
**Pull request by MacJIOFF**
A notification that a new item is being recorded has been added to the "Partners" lookup form. In the form of selection of the reference book "Partners" added processing of notification and positioning in the list.

### CODE REVIEW

File: IRP/src/Catalogs/Partners/Forms/ItemForm/Module.bsl [hunk 1]
```diff
@@ -1,5 +1,13 @@
+
 #Region FormEvents
 
+&AtClient
+Procedure AfterWrite(WriteParameters)
+	If ThisIsNew Then
+		Notify("WritingNew", Object.ref);
```
Comment (DitriXNew) [inline IRP/src/Catalogs/Partners/Forms/ItemForm/Module.bsl:7]: change "ref" to "Ref"

File: IRP/src/Catalogs/Partners/Forms/ChoiceForm/Module.bsl [hunk 2]
```diff
@@ -67,4 +67,13 @@ EndProcedure
 Procedure SetBooleanListFilter(FilterItems, FieldName, RightValue)
 	CommonFunctionsClientServer.SetFilterItem(FilterItems, FieldName, RightValue, DataCompositionComparisonType.Equal,
 		RightValue = True);
+EndProcedure
+
+&AtClient
+Procedure NotificationProcessing(EventName, Parameter, Source)
+	If EventName = "WritingNew"
```
Comment (DitriXNew) [inline IRP/src/Catalogs/Partners/Forms/ChoiceForm/Module.bsl:74]: Rename event name "WritingNew" to "NewPartnerCreated"

File: IRP/src/Catalogs/Partners/Forms/ItemForm/Form.form [hunk 3]
```diff
@@ -1303,6 +1307,23 @@ WHERE
       <getInvisibleFieldPresentations>true</getInvisibleFieldPresentations>
     </extInfo>
   </attributes>
+  <attributes>
+    <name>ThisIsNew</name>
+    <title>
+      <key>ru</key>
```
Comment (DitriXNew) [inline IRP/src/Catalogs/Partners/Forms/ItemForm/Form.form:1313]: Change "ru" to "en"

**Review by DitriXNew — CHANGES_REQUESTED** [review]

File: IRP/src/Catalogs/Partners/Forms/ItemForm/Module.bsl (hunk 1 above)
Comment (MacJIOFF) [reply IRP/src/Catalogs/Partners/Forms/ItemForm/Module.bsl]: +

File: IRP/src/Catalogs/Partners/Forms/ChoiceForm/Module.bsl (hunk 2 above)
Comment (MacJIOFF) [reply IRP/src/Catalogs/Partners/Forms/ChoiceForm/Module.bsl]: +

File: IRP/src/Catalogs/Partners/Forms/ItemForm/Form.form (hunk 3 above)
Comment (MacJIOFF) [reply IRP/src/Catalogs/Partners/Forms/ItemForm/Form.form]: done

**Review by DitriXNew — APPROVED** [review]

**Comment by DitriXNew** [comment]
@seramiti надо тесты обновить на этот функционал,

**Pull request merged**